src/modality-specific-files/physiological-and-other-continuous-recordings.md

```diff
@@ -18,40 +9,339 @@ specified using two files:
+TSVGZ files MUST be accompanied by a JSON file with the same name as their
+corresponding tabular file but with a `.json` extension.
+Additional [metadata corresponding to TSVGZ files](../common-principles.md#tabular-files)
+MAY be included as prescribed by the common principles.
+For example, indicating units for recorded time series.
+
+??? warning "Columns of TSVGZ files MUST be defined in the corresponding JSON sidecar and the tabular content MUST NOT include a header line."
+
+    In contrast to plain-text TSV files in BIDS, TSVGZ files MUST NOT
+    include a header line.
+    Column names MUST be specified in the JSON file following the
+    [`Columns` metadata](../glossary.md#columns-metadata) specifications.
+    As for all [tabular data structures](../common-principles.md#tabular-files),
+    column names MUST NOT be blank (that is, an empty string),
+    and MUST NOT be duplicated within a single JSON file describing a headerless
+    TSV file.
+
+    TSVGZ are headerless to improve compatibility with existing software
+    (for example, FSL, PNM), and to facilitate the support for other file formats
+    in the future.
+
+Summary template:
+
+<div class="highlight">
+<pre><code><a href="../appendices/entities.html#sub">sub-&lt;label&gt;</a>/
+    [<a href="../appendices/entities.html#ses">ses-&lt;label&gt;</a>/]
+        &lt;<a href="../glossary.html#data_type-common_principles">datatype</a>&gt;/
+            &lt;matches&gt;[_<a href="../appendices/entities.html#recording">recording</a>-&lt;<a href="../glossary.html#label-common_principles">label</a>&gt;]_&lt;<a href="../glossary.html#physio-suffixes">physio</a>|<a href="../glossary.html#physioevents-suffixes">physioevents</a>|<a href="../glossary.html#stim-suffixes">stim</a>&gt;<a href="../glossary.html#json-extensions">.json</a>
+            &lt;matches&gt;[_<a href="../appendices/entities.html#recording">recording</a>-&lt;<a href="../glossary.html#label-common_principles">label</a>&gt;]_&lt;<a href="../glossary.html#physio-suffixes">physio</a>|<a href="../glossary.html#physioevents-suffixes">physioevents</a>|<a href="../glossary.html#stim-suffixes">stim</a>&gt;<a href="../glossary.html#tsvgz-extensions">.tsv.gz</a>
+</code></pre></div>
+
+For the template directory name, `<datatype>` can correspond the following
+[data types](../glossary.md#data_type-common_principles):
+`beh`, `dwi`, `eeg`, `func`, `ieeg`, `meg`, `motion`, `nirs`, `perf`, or `pet`.
+In the template filenames, the `<matches>` part corresponds to those entities
+before the suffix that identify the reference run.
+For example, for the file `sub-01_task-nback_run-1_bold.nii.gz`,
+`<matches>` would correspond to `sub-01_task-nback_run-1`.
+Note that when supplying a `<matches>_<physio|physioevents|stim>.tsv.gz` file,
+an accompanying `<matches>_<physio|physioevents|stim>.json` MUST be supplied as well.
+
+<details class="tip">
+<summary>Expanded filename template</summary>
```

Comment:
same here: not sure the pdf build will like this.

Reply:
ignore my comment if this just WIP

Reply:
Ugh, right. Let's punt this for when the meat is ready.